pt_BR.ini

```diff
@@ -315,7 +317,8 @@ Remove From Recent = Remover dos recentes
 SaveData = Dados de gravação do jogo
 Show In Folder = Mostrar na pasta
 USA = USA
-Use UI background = Usar UI background
+Use UI background = Usar background
```

Comment:
In portuguese "UI" don't have a translation and looks much tecnical so I removed, I would put "Usar imagem do jogo na interface" wich means: Use game image in interface but as you said that can make the button bad.